README.md

```diff
@@ -1,44 +1,52 @@
 # UMA Protocol
 
-## Contact
+<div style="text-align:center"><img src="./documentation/Logo.png" /></div>
 
-- [Slack](https://umaprotocol.slack.com): to join, use this
-  [invite link](https://join.slack.com/t/umaprotocol/shared_invite/enQtNTk4MjQ4ODY0MDA1LTM4ODg0NGZhYWZkNjkzMDE4MjU0ZGFlYWQzZTFiZWFlZjI2NDE4OGI2NWY3OTdhYjYyZjg0MjAzMTgwODVhZTE).
-  Please use Slack for all technical questions and discussions.
-- [Email](mailto:hello@umaproject.org): for anything non-technical.
+[![<UMAprotocol>](https://circleci.com/gh/UMAprotocol/protocol.svg?style=shield)](https://app.circleci.com/pipelines/github/UMAprotocol/protocol)
+![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/umaprotocol/voting)
 
-## Documentation
+![GitHub](https://img.shields.io/github/license/UMAprotocol/protocol)
+![GitHub last commit](https://img.shields.io/github/last-commit/UMAprotocol/protocol)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/UMAprotocol/protocol)
+![GitHub contributors](https://img.shields.io/github/contributors-anon/UMAprotocol/protocol)
```

Comment:
Maybe link to our contributor page?

Reply:
done

Reply:
We could also add contributor guidelines to our repo if we wanted at some point? Right now there are no instructions for community members on how they should contribute code if they wanted to.

Reply:
Agreed, probably not a big deal until we get more community contribution interest, though.